File: GandanBusJava/app/src/main/java/com/gandan/android/gandanbusjava/model/BusLocationList.java
package com.gandan.android.gandanbusjava.model;

import org.simpleframework.xml.Element;
import org.simpleframework.xml.ElementList;


public class BusLocationList {

    @Element(name = "endBus")
    private String endBus;

    @Element(name = "plateType")
    private String plateType;

    @Element(name = "remainSeatCnt")
    private String remainSeatCnt;

    @Element(name = "stationSeq")
    private String stationSeq;

    @Element(name = "stationId")
    private String stationId;

    @Element(name = "routeId")
    private String routeId;

    @Element(name = "lowPlate")
    private String lowPlate;

    @Element(name = "plateNo")
    private String plateNo;

    public String getEndBus ()
    {
        return endBus;
    }

    public void setEndBus (String endBus)
    {
        this.endBus = endBus;
    }

    public String getPlateType ()
    {
        return plateType;
    }

    public void setPlateType (String plateType)
    {
        this.plateType = plateType;
    }

    public String getRemainSeatCnt ()
    {
        return remainSeatCnt;
    }

    public void setRemainSeatCnt (String remainSeatCnt)
    {
        this.remainSeatCnt = remainSeatCnt;
    }

    public String getStationSeq ()
    {
        return stationSeq;
    }

    public void setStationSeq (String stationSeq)
    {
        this.stationSeq = stationSeq;
    }

    public String getStationId ()
    {
        return stationId;
    }

    public void setStationId (String stationId)
    {
        this.stationId = stationId;
    }

    public String getRouteId ()
    {
        return routeId;
    }

    public void setRouteId (String routeId)
    {
        this.routeId = routeId;
    }

    public String getLowPlate ()
    {
        return lowPlate;
    }

    public void setLowPlate (String lowPlate)
    {
        this.lowPlate = lowPlate;
    }

    public String getPlateNo ()
    {
        return plateNo;
    }

    public void setPlateNo (String plateNo)
    {
        this.plateNo = plateNo;
    }

}

File: GandanBus/app/src/main/java/com/gandan/android/gandanbus/model/BusLocationList.kt
package com.gandan.android.gandanbus.model

import com.tickaroo.tikxml.annotation.PropertyElement
import com.tickaroo.tikxml.annotation.Xml
import org.simpleframework.xml.Element
import org.simpleframework.xml.Path
import org.simpleframework.xml.Root

@Root
class BusLocationList {
    @Path("busLocationList")
    @field:Element(name = "endBus") var endBus : Int? = 0

    @field:Element(name = "plateType") var plateType : Int? = 0

    @field:Element(name = "remainSeatCnt") var remainSeatCnd : Int? = 0

    @field:Element(name = "stationSeq") var stationSeq : Int? = 0

    @field:Element(name = "stationId") var stationId : Long? = 0

    @field:Element(name = "routeId") var routeId : Long? = 0

    @field:Element(name = "lowPlate") var lowPlate : Int? = 0

    @field:Element(name = "plateNo") var plateNo : String? = null
}